packages/origin-247-transfer/src/EnergyTransferRequest.ts

```diff
@@ -102,9 +102,15 @@ export class EnergyTransferRequest {
         }
 
         if (currentStatus !== TransferValidationStatus.Pending) {
-            throw new Error(
-                `Cannot update status of transfer request: ${this.attrs.id} to ${status}, because it already has status ${currentStatus}`
+            /**
+             * This may happen randomly if for example we have asymmetric validator,
+             * that instantly updates validation status after we start validation
+             */
+            console.warn(
```

Comment:
Does it make sense to use NestJS's `logger.warn()` instead of console?

Reply:
I considered this, but this is domain object, so we would have to create it with logger in constructor. Does this make sense?

Reply:
If you find a better way, great. Otherwise, I don't mind this too much.

Reply:
It could just mean that if we add some custom middleware to the NestJS logger, like logging to a file or something, this log might not get stored.

Reply:
Let me correct this quickly somehow :)

Reply:
Please take a look now